### PR TITLE
Do not show hidden github comments

### LINKED ISSUE #1640: Please, do not show hidden cluster ticket comments
**What would you like to be added**:
When tickets are shown for a cluster, please do not show hidden comments. There were (probably always) good reasons to hide a comment in GitHub, so they shouldn't be shown in the Dashboard either.

**Why is this needed**:
Show only appropriate content. Respect decision made for GitHub also in Dashboard.

### PR DESCRIPTION
**Pull request by holgerkoser**
**What this PR does / why we need it**:
This PR introduces functionality to retrieve GitHub comments using the GitHub [GraphQL API](https://docs.github.com/en/graphql/reference/objects#issuecomment). It leverages the `isMinimized` property, unique to the GraphQL API's IssueComment object, to filter out minimized comments. As a result, only non-minimized comments will be displayed on the dashboard.

**Which issue(s) this PR fixes**:
Fixes #1640

**Special notes for your reviewer**:

**Release note**:

```feature user
Hidden GitHub comments will no longer be displayed on the Cluster Details page.
```

### CODE REVIEW

File: backend/lib/github/index.js [hunk 1]
```diff
@@ -48,13 +48,53 @@ function closeIssue ({ number }) {
...
-    owner,
-    repo,
-    issue_number: number
-  })
-  return octokit.paginate(options)
+async function getComments ({ number }) {
+  const query =
+  `query paginate($cursor: String) {
+    repository(owner: "${owner}", name: "${repo}") {
+      issue(number: ${number}) {
```
Comment (holgerkoser) [inline backend/lib/github/index.js:55]: Escape all params

Comment (holgerkoser) [reply backend/lib/github/index.js]: ✔️

**Review by grolu — APPROVED** [review]
/lgtm

**Review by petersutter — APPROVED** [review]
/lgtm

**Pull request merged**